src/mame/video/konamiic.txt

```diff
@@ -14,16 +14,16 @@ TODO:
   Games requiring shadows to NOT affect sprites behind them:
   - Asterix (Asterix's shadow would be over his feet otherwise)
   - X-Men is dubious, see enemies halfway through level 1 coming from above with
-    boulders over their heads.
+	boulders over their heads.
 
 - scrollcontrol = 30 in Golfing Greats (leader board)
 
 
 
-                      Emulated
-                         |
-                  board #|year    CPU      tiles        sprites  priority palette    other
-                    -----|---- ------- ------------- ------------- ------ ------ ----------------
+					  Emulated
+						 |
+				  board #|year    CPU      tiles        sprites  priority palette    other
+					-----|---- ------- ------------- ------------- ------ ------ ----------------
```

Comment:
All the changes to this file seem to be in leading whitespace – is this file supposed to be changed at all?

Reply:
no it isn't, it just gets picked up every time I run srcclean, and I forgot to exclude it this time